packages/manager/modules/dbaas-logs/src/logs/empty/empty.html

```diff
@@ -0,0 +1,52 @@
+<div class="container mx-0 px-0 ldp-project-empty">
+    <div class="row align-items-center">
+        <div class="col-md-4">
+            <div data-ng-class="{ 'placeholder p-5' : !$ctrl.imageSource }">
+                <img
+                    data-ng-src="{{:: $ctrl.imageSource || $ctrl.placeholder }}"
+                    alt=""
```

Comment:
?
```suggestion
                    alt="{{$ctrl.imageAlt}}" 
```

Reply:
If the image has only some decorative purpose and don't provide further information, it's ok to leave an empty alt (as long as it's there)

Reply:
yes, it has been removed